### PR TITLE
emotion for stats page

### LINKED ISSUE #1050: pages/stats CSS into emotion

### PR DESCRIPTION
**Pull request by sastels**
Resolves #1050

### CODE REVIEW

File: pages/stats.js [hunk 1]
```diff
@@ -2,53 +2,41 @@ import React, { Component } from "react";
...
-const styles = theme => ({
-  marginTop: {
-    marginTop: theme.spacing.unit * 3
-  },
-  table: {
-    // width: 1000
-  }
-});
+const marginTop = css`
+  margin-top: 24px;
```
Comment (apmcguigan) [inline pages/stats.js]: we moved some commonly used values over to theme.js (1px off, feel free to change it)

try:
`import { globalTheme } from "../theme";`
and then
`margin-top: ${theme.marginTop};`

Comment (sastels) [reply pages/stats.js]: nice!

Comment (sastels) [reply pages/stats.js]: lol actually `theme.spacing.unit` was 24, I checked it via the browser inspector before fixing the value here. I think some places had a `+1` added to it though? either way, 25 is fine.

**Review by apmcguigan — APPROVED** [review]
Awesome!

**Pull request merged**